lib/rules/selector-max-type/__tests__/index.js

```diff
@@ -549,3 +549,82 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
+  ruleName,
+  config: [0, { ignore: ["next-sibling"] }],
+  skipBasicChecks: true,
+  syntax: "scss",
+
+  accept: [
+    {
+      code: ".foo + a {}"
+    },
+    {
+      code: ".foo, .bar + a {}"
+    },
+    {
+      code: ".foo + a + b {}"
+    },
+    {
+      code: ".foo { .bar + a {} }"
+    },
+    {
+      code: ".foo, .bar + baz {}"
+    },
+    {
```

Comment:
I think we can remove this test. It doesn't add much and I believe the trailing comma after the closing brace makes it invalid CSS.